lib/huginn_acumen_product_agent/acumen_product_agent.rb

```diff
@@ -95,13 +160,15 @@ def receive(incoming_events)
         private
 
         def handle(event)
+            # Process agent options
             endpoint = interpolated['endpoint']
             site_code = interpolated['site_code']
             password = interpolated['password']
             physical_formats = interpolated['physical_formats']
```

Comment:
Because it was defined previously, and it's not exactly hurting anything to leave it in.

Given that we are still differentiating between physical & digital formats, I didn't want to remove anything related to that distinction in case testing/QA ultimately required us to put it back in.